app/api/v1/airline/[airlineId]/route.ts

```diff
@@ -0,0 +1,316 @@
+import { NextRequest, NextResponse } from "next/server"
+
+import { getDatabase } from "@/lib/couchbase-connection"
+import { TAirline,AirlineSchema } from "@/app/models/Airline"
+import { DocumentNotFoundError,DocumentExistsError } from 'couchbase';
+
+import { ZodError } from "zod";
+
+/**
+ * @swagger
+ * /api/v1/airline/{airlineId}:
+ *   get:
+ *     summary: Get an airline by ID
+ *     description: |
+ *       Get Airline with specified ID.
+ *       
+ *       This provides an example of using [Key Value operations](https://docs.couchbase.com/nodejs-sdk/current/howtos/kv-operations.html) in Couchbase to get a document with specified ID.
+ *       
+ *       Key Value operations are unique to Couchbase and provide very high-speed get/set/delete operations.
+ *       
+ *       Code: `airline/[airlineId]/route.ts` Method: `GET`
+ *     tags:
+ *        - Airline
+ *     parameters:
+ *       - name: airlineId
+ *         in: path
+ *         description: ID of the airline
+ *         required: true
+ *         schema:
+ *           type: string
+ *     responses:
+ *       200:
+ *         description: Returns the airline
+ *       404:
+ *         description: Airline not found
+ *       500:
+ *         description: An error occurred while fetching airline
+ */
+export async function GET(
+  req: NextRequest,
+  { params }: { params: { airlineId: string } }
+) {
+  try {
+    const { airlineId } = params
+    const { airlineCollection } = await getDatabase()
+
+    const airline = await airlineCollection.get(airlineId)
+    return NextResponse.json(airline.content as TAirline, { status: 200 })
+  } catch (error) {
+    if (error instanceof DocumentNotFoundError) {
+      return NextResponse.json(
+        { message: "Airline not found", error: "Airline not found" },
+        { status: 404 }
+      )
+    } else {
+      return NextResponse.json(
+        {
+          message: "An error occurred while fetching airline",
+        },
+        { status: 500 }
+      )
+    }
+  }
+}
+
+/**
+ * @swagger
+ * /api/v1/airline/{airlineId}:
+ *   post:
+ *     summary: Create an airline
+ *     description: |
+ *       Create an airline with specified ID.
+ *       
+ *       This provides an example of using [Key Value operations](https://docs.couchbase.com/nodejs-sdk/current/howtos/kv-operations.html) in Couchbase to create a document with specified ID.
+ *       
+ *       Key Value operations are unique to Couchbase and provide very high-speed get/set/delete operations.
+ *       
+ *       Code: `airline/[airlineId]/route.ts` Method: `POST`
+ *     tags:
+ *        - Airline
+ *     parameters:
+ *       - name: airlineId
+ *         in: path
+ *         description: ID of the airline
```

Comment:
Same as above.
Similarly please do it for other routes as well.

Reply:
added examples in all routes